src/main/java/teammates/common/datatransfer/questions/FeedbackMcqQuestionDetails.java

```diff
@@ -37,6 +37,7 @@
     private int numOfMcqChoices;
     private List<String> mcqChoices;
     private boolean otherEnabled;
+    private boolean generatedOptionsEnabled;
```

Comment:
I don't think you should change this. This would have impact to the DB.

Reply:
I am not sure how to use the checkbox in the frontend otherwise. 
Workflow:-
1) If the user clicks the checkbox, then discards the change, the checkbox is still ticked. Which is why I included this

Reply:
> If the user clicks the checkbox

Which checkbox?

Reply:
<img width="398" alt="Screenshot 2019-03-12 at 8 05 39 PM" src="https://user-images.githubusercontent.com/28557842/54198802-5633d780-4502-11e9-98ea-15726a70894c.png">

Reply:
Why not make the condition the same as whatever condition makes it unticked in the beginning?

Reply:
> If the user clicks the checkbox, then discards the change, the checkbox is still ticked.

I would consider it as a bug in V6.

Reply:
>Why not make the condition the same as whatever condition makes it unticked in the beginning? 
I am not sure how to listen to event when discard is clicked, and update the checkbox in the Mcq component

Reply:
@xpdavid the workflow in v6 is fine, I was just using the workflow as an example here

Reply:
>Why not make the condition the same as whatever condition makes it unticked in the beginning

Fixed this in latest commit